packages/genui/a2ui-playground/src/styles.css

```
@@ -0,0 +1,1027 @@
+  --geist-code-fg: #e8e8e8;
+  --geist-shadow-sm:
+    0 1px 2px rgba(0, 0, 0, 0.04), 0 1px 3px rgba(0, 0, 0, 0.06);
+  --geist-shadow-md: 0 4px 14px rgba(0, 0, 0, 0.08);
+  --geist-shadow-lg: 0 8px 30px rgba(0, 0, 0, 0.12);
+  --geist-radius-sm: 6px;
+  --geist-radius-md: 8px;
+  --geist-radius-lg: 12px;
+  --geist-radius-xl: 16px;
+  --geist-transition: 150ms cubic-bezier(0.4, 0, 0.2, 1);
+  --geist-font:
+    -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu,
```

Stylelint (17.7.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)